forge-ai/src/main/java/forge/ai/AiController.java

```diff
@@ -1376,29 +1375,10 @@ public List<SpellAbility> chooseSpellAbilityToPlay() {
                 Card land = chooseBestLandToPlay(landsWannaPlay);
                 if ((!player.canLoseLife() || player.cantLoseForZeroOrLessLife() || ComputerUtil.getDamageFromETB(player, land) < player.getLife())
                         && (!game.getPhaseHandler().is(PhaseType.MAIN1) || !isSafeToHoldLandDropForMain2(land))) {
-                    final List<SpellAbility> abilities = Lists.newArrayList();
-
-                    // TODO extend this logic to evaluate MDFC with both sides land
```

Comment:
can we keep this todo? it's still valid

Reply:
MDFC should be handled with `land.getAllPossibleAbilities(player, true);`?

Reply:
yea but if the card has two LandAbility, e.g. _Darkbore Pathway_ AI will just always take the first one